src/const.h

```diff
@@ -519,8 +519,13 @@ enum PlayerFlags : uint64_t {
 
 #define CHANNEL_GUILD 0x00
 #define CHANNEL_PARTY 0x01
+
+#define CHANNEL_CAST 0xFFFE
```

Comment:
I haven't had much time lately and I'm a little out of date, is 10.80+ support already in master?